website/static/js/addProjectPlugin.js

```diff
@@ -89,6 +96,15 @@ var AddProject = {
             var success = function _success (result) {
                 self.viewState('success');
                 self.goToProjectLink(result.data.links.html);
+                if (self.newProjectInstitutions.length > 0){
+                    var newNodeApiUrl = $osf.apiV2Url('nodes/' + result.data.id + '/relationships/institutions/');
+                    var data = {data: self.newProjectInstitutions.filter(
+                        function(id){return self.checkedInstitutions[id];}
+                    ).map(
+                        function(id){return {type: 'institutions', id: id};}
+                    )};
+                    m.request({method: 'POST', url: newNodeApiUrl, data: data, config: xhrconfig});
```

Comment:
Also, what happens if the user navigates away from the page before this request finishes?

It might be better to only show the success alert once both requests have finished. You can do this by chaining promises.

```js
// send create project request,
var request = m.request(....)
if user has selected institutions:
    request.then(sendAffiliationRequest)
request.then(handleSuccess, handleError)
````